src/pages/ThorChainLP/components/ReusableLpStatus/TransactionRow.tsx

```diff
@@ -521,24 +533,17 @@ export const TransactionRow: React.FC<TransactionRowProps> = ({
             if (!inboundAddressData) throw new Error('No inboundAddressData found')
             // ATOM/RUNE/ UTXOs- obviously make me a switch case for things to be cleaner
             if (!accountAssetAddress) throw new Error('No accountAddress found')
-
-            const memo = `+:${thorchainNotationAssetId}:${otherAssetAddress ?? ''}:ss:${
-              confirmedQuote.feeBps
-            }`
-
-            const estimateFeesArgs = {
-              cryptoAmount: amountCryptoPrecision,
-              assetId,
-              to: inboundAddressData?.address,
-              from: accountAssetAddress,
-              sendMax: false,
-              memo,
-              accountId,
-              contractAddress: undefined,
-            }
+            if (!estimateFeesArgs) throw new Error('No estimateFeesArgs found')
+            const _amountCryptoPrecision = isDeposit
```

Comment:
And here too

Reply:
https://github.com/shapeshift/web/pull/6285/commits/32f59bac73d2f81b3cd1f84ddbbf2c6a3244aa8a